lib/hammer_cli_foreman/operating_system.rb

```diff
@@ -191,7 +191,7 @@ def base_action_params
         {"operatingsystem_id" => option_id}
       end
 
-      build_options
+      build_options expand: { except: %i[organizations locations] }, without: %i[organization_id location_id]
```

Comment:
You probably want to do that for `SetOsDefaultTemplate` command as well :)

Reply:
Done :)